GoToProject/src/components/CustomMapView/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { FlexContainer, InputLabel } from '../styled';
+import MapView, {Marker, PROVIDER_GOOGLE} from 'react-native-maps';
+import {Dimensions} from 'react-native';
+
+
+const {width, height} = Dimensions.get('window');
+
+const ASPECT_RATIO = width / height;
+const LATITUDE_DELTA = 0.0922;
+const LONGITUDE_DELTA = 0.0421;
+
+const GOOGLE_MAPS_APIKEY = 'AIzaSyB4SwDlMm4Dr2c7j5A-iRbFbAWxIKO-tOA';
+
+export const CustomMapView = ({ mapHeight, mapWidth, siteLatitude, siteLongitude, siteName }) => {
+
+const siteLocation = [
+    {
+      title: siteName,
+      coordinates: {
+        latitude: siteLatitude,
+        longitude: siteLongitude,
+        latitudeDelta: LATITUDE_DELTA,
+        longitudeDelta: LONGITUDE_DELTA,
+      },
+    },
+  ];
+
+const placeMarker = siteLocation.map(place => (
+  
+    <Marker key={place.title}
+    coordinate={{
+      latitude: place.coordinates.latitude,
+      longitude: place.coordinates.longitude,
+    }}
+    title={place.title} />
```

Comment:
Can you call this component from an external file?

Reply:
component refactored